Source/Core/Core/Boot/Boot.cpp

```diff
@@ -255,7 +255,8 @@ bool CBoot::BootUp()
   g_symbolDB.Clear();
 
   // PAL Wii uses NTSC framerate and linecount in 60Hz modes
-  VideoInterface::Preset(_StartupPara.bNTSC || (_StartupPara.bWii && _StartupPara.bPAL60));
+  bool NTSC = DiscIO::IsNTSC(_StartupPara.m_region);
```

Comment:
This should probably be left in the conditional, or both be moved to a new function. This variable is only ever used in one place (it's also non-const).